### PR TITLE
support bernoulli opset 15

### PR DESCRIPTION
**Pull request by chudegao**
Signed-off-by: degaochu <degaochu@cn.ibm.com>

This operator is non-deterministic and may not produce the same values so we still need to exlucde the test case.

### CODE REVIEW

File: onnx_tf/handlers/backend/bernoulli.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+@ps_description(
+    "Bernoulli with float type seed will be converted to int type seed")
+class Bernoulli(BackendHandler):
+
+  @classmethod
+  def _common(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    dtype = x.dtype
+    if 'dtype' in node.attrs:
+      dtype = node.attrs.get('dtype')
```
Comment (chinhuang007) [inline onnx_tf/handlers/backend/bernoulli.py:21]: Could be combined to one line, dtype = node.attrs.get("dtype", x.dtype)

**Comment by chinhuang007** [comment]
Everything looks good. Can you re-apply cla?

File: onnx_tf/handlers/backend/bernoulli.py (hunk 1 above)
Comment (chudegao) [reply onnx_tf/handlers/backend/bernoulli.py]: Thanks, updated.

**Comment by chudegao** [comment]
> Everything looks good. Can you re-apply cla?

I'm not sure how to handle this. Can you update status manually. I hit this issues before. Sometimes, it will pass the check after serveral hours. Sometimes it keeps waiting. 

图片

**Review by chinhuang007 — APPROVED** [review]
LGTM

**Pull request merged**